packages/core/Tensei.ts

```diff
@@ -498,7 +498,7 @@ export class Tensei implements TenseiContract {
   }
 
   public mailer(driver: string) {
-    this.mailerConfig.mailer = (driver as unknown) as never
+    this.mailerConfig.mailer = driver as unknown as never
```

Comment:
A lot of formatting changes from your editor. Please run `yarn prettier` at the root to fix all formatting issues.